articles/libraries/auth0-swift/index.md

```diff
@@ -77,7 +79,30 @@ Auth0
    }
 ```
 
-#### Passwordless Login
+### Passwordless Login
+
+Logging in with a Passwordless is slightly different. Passwordless can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. 
+
+#### Passwordless Parameters
+
+As you can see, Passwordless authentication can be started with a variety of different parameters. 
+
+```
+.startPasswordless(email/phoneNumber: String, type: String, connection: String)
+```
+
+- **Parameter One** - Either the parameter `email` or `phoneNumber`, depending on which you intend to use. The value of either should be a string.
+- **Parameter Two** - The parameter `type`, its value should be either `.Code` or `.iOSLink`. The default is `.Code` (a code is sent to the user, then they input it in a secondary screen), but if you have iOS Universal Links configured, you can use `.iOSLink`.
```

Comment:
Is there any interal resource to link to iOS Universal Links? Can we link to Apple docs? https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html